manual.md

```diff
@@ -52,31 +52,31 @@ Before a RGP can be submitted on-chain, it must first complete **an off-chain r
 The ***Discussion*** phase is used to review the first draft of new proposals. This is the primary period for the community to review and provide feedback on proposals drafts. The more feedback provided during this phase allows for more detailed and refined proposals to vote on in later stages.
 
 When preparing the proposal drafts for the _Discussion_ phase, proposal authors should:
-- Use [these templates](https://github.com/radicle-foundation/radworks-governance/tree/main/templates) to create their proposals
+- Use [these templates](https://github.com/radicle-foundation/radworks-governance/tree/main/templates) to create their proposals. Please see the 'Proposal Types' section below to determine which template to use. 
 - Try to include as much detail as possible in this first draft of their proposals
-- Post their proposal drafts under [Proposal Drafts](https://community.radworks.org/c/governance/discussions/19) on the forum by the _second Monday_ of each cycle (see 'Proposal Cycles' for details).  
```

Comment:
Open question: should we change the deadline for posting to the first wednesday of the cycle? seems like people wait until the last moment and then complain about there not being enough time for feedback to be included so maybe we should just change the requirement